pythonforandroid/recipes/numpy/__init__.py

```diff
@@ -40,10 +40,10 @@ def build_arch(self, arch):
         super().build_arch(arch)
         self.restore_hostpython_prerequisites(["cython"])
 
-    def get_hostrecipe_env(self, arch):
-        env = super().get_hostrecipe_env(arch)
+    def get_hostrecipe_env(self, arch=None):
+        env = super().get_hostrecipe_env(arch=arch)
         env['RANLIB'] = shutil.which('ranlib')
-        env["LDFLAGS"] += " -lm"
+        env["LDFLAGS"] = " -lm"
```

Comment:
Is it expected that we override it fully rather than appending?

Reply:
I totally forgot why I did that, but now I'm thinking we don't even need to append anything.